lib/prelude.ml

```diff
@@ -29,3 +29,4 @@ let print = Io.print
 
 let flip f x y = f y x
 let x |> f = f x
+let f # g = fun x -> f (g x)
```

Comment:
Hrrm, is this not the same as `%>` in data/category.ml?

Reply:
But specialised to `(->)` so we don't need `Category` dictionaries